torch/nn/functional.py

```diff
@@ -3829,26 +3829,125 @@ def fold(input, output_size, kernel_size, dilation=1, padding=0, stride=1):
 def _pad_circular(input, padding):
     # type: (Tensor, List[int]) -> Tensor
     """
-    Arguments
-        :param input: tensor of shape :math:`(N, C_{\text{in}}, H, [W, D]))`
-        :param padding: (tuple): m-elem tuple where m is the degree of convolution
-    Returns
-        :return: tensor of shape :math:`(N, C_{\text{in}}, [D + 2 * padding[0],
-                 H + 2 * padding[1]], W + 2 * padding[2]))`
+    Args:
+        input: Tensor that follows the formatting of the input to convolution
+            layers.
+        padding: Tuple with length two times the degree of the convolution. The
+            order of the integers in the tuple are shown in the following
+            example:
+
+            For 3D convolutions:
```

Comment:
This should enumerate all cases handled: NCHW and NCDHW.

Reply:
I removed this part of the docstring since it looks really confusing at first sight, and would be too verbose IMO if it was described for each case. Instead, the examples should help illuminate how to form the padding parameter. Also, the main function F.pad describes this in their documentation, and _pad_circular is a private function anyway meant to be called through F.pad. Let me know if this is ok, if not I'll fix it!